pygmt/tests/test_datasets_samples.py

```diff
@@ -145,3 +147,17 @@ def test_hotspots():
         "place_name",
     ]
     assert isinstance(data, pd.DataFrame)
+
+
+def test_load_static_earth_relief():
+    """
+    Check that @static_earth_relief.nc loads without errors.
+    """
+    data = load_sample_data("static_earth_relief")
+    assert data.dims == ("lat", "lon")
+    assert data.shape == (14, 8)
+    assert data.min() == 190
+    assert data.max() == 981
+    assert data.median() == 467
+    npt.assert_allclose(data.mean(), 490.87946)
```

Comment:
```suggestion
```
I don't think this adds much to the other `min, max, and median` assertions, so we could leave it out to cut down on imports.